dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -601,10 +602,63 @@ public Response saveLayout(
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     @Path("{pageId}/content")
+    @Operation(operationId = "postPageContent",
+        summary = "Updates contentlets in a page's containers",
+        description = "This method updates a page to reflect the addition (or removal) of " + 
+                        "contentlets to (or from) containers. It takes a JSON object that " +
+                        "returns variables associated with the contentlets and their modifications.",
+        tags = {"Page"},
+        responses = {
+                @ApiResponse(responseCode = "200", description = "Content updated successfully",
+                        content = @Content(mediaType = "application/json",
+                                schema = @Schema(implementation = ResponseEntityStringView.class),
```

Comment:
it is interesting that you are not returning this as part of the method, 
you can avoid the schema def, if return the actual ResponseEntityStringView instead of the Response